settings.gradle

```diff
@@ -58,6 +59,8 @@ if (isEclipse) {
 
 include projects.toArray(new String[0])
 
+project(':build-tools').projectDir = new File(rootProject.projectDir, 'buildSrc')
```

Comment:
I don't want to use the name buildSrc for the artifact because I think it is confusing. It is only called buildSrc in the project because this is required by gradle.